src/signin.js

```diff
@@ -1303,6 +1304,19 @@ async function getUserRoles () {
   return UI.store.each(profile, ns.rdf('type'), null, preferencesFile.doc())
 }
 
+function findProfileImage (profile) {
```

Comment:
I think moving ths code from buttons.setImage wasa n odd move.  The setImage function in more general and lower level. It should be called by setProfileImage which is specific to a profile image. The setImage can be used by other things as well as profile image — images of people or organizations, and we can add to it more image predicates with time like from schema.org

Reply:
My thinking was that findProfileImage belongs to other user-related functions - where would you move it? Or would you make this function reusable in another way?

Reply:
Me and @timbl had a chat outside of this - we concluded that we will move findProfileImage back to buttons, and rename it findImage (since it's a more general thing than just connected to profiles).

We also concluded to create https://github.com/solid/solid-ui/issues/120 since we at some point want to clean up the API a bit to make it more intuitive.